Tools/Create-Azure-Sentinel-Solution/input/Solution_DigitalShadowsSearchlight.json

```
@@ -0,0 +1,15 @@
+ {
+    "Name": "DigitalShadowsSearchlight",
+    "Author": "{Angus Gordon - angus.gordon@digitalshadows.com}",
```

```suggestion
    "Author": "Digital Shadows - support@digitalshadows.com",
```

Definitely don't want the curly-braces on here